doc/howto/write_a_view/solution_view.cpp

```diff
@@ -49,12 +49,12 @@ class my_iterator : public seqan3::detail::inherited_iterator_base<my_iterator<u
 };
 
 // The inherited_iterator_base creates the necessary code so we also model RandomAccess now!
```

Comment:
```suggestion
// The inherited_iterator_base creates the necessary code so we also model random_access now!
```

?

Reply:
Well it should have been named like the concept :man_shrugging: